### PR TITLE
Add: missing preprocessFileLineNumbers part2

### PR DESCRIPTION
**Pull request by 1kuemmel1**
- Ignore changelog.

**When merged this pull request will:**
- similar like #422 
- replace `preprocessFile ` with `preprocessFileLineNumbers`
- replace `waitUntil ` with `CBA_fnc_waitUntilAndExecute` (just in [init_server.sqf](https://github.com/1kuemmel1/HeartsAndMinds/blob/f38f450852bc0f6b561d7958c2586b64a7897d02/%3DBTC%3Dco%4030_Hearts_and_Minds.Altis/core/init_server.sqf#L20))

**Final test:**
- [x] local
- [x] server

### CODE REVIEW

File: =BTC=co@30_Hearts_and_Minds.Altis/init.sqf [hunk 1]
```diff
@@ -2,19 +2,19 @@ enableSaving [false,false];
...
 if (hasInterface) then {btc_intro_done = [] spawn btc_fnc_intro;};
 
-call compile preprocessFile "core\def\mission.sqf";
-call compile preprocessFile "define_mod.sqf";
+call compile preprocessFileLineNumbers "core\def\mission.sqf";
+call compile preprocessFileLineNumbers "define_mod.sqf";
 
 if (isServer) then {
-	call compile preprocessFile "core\init_server.sqf";
+		call compile preprocessFileLineNumbers "core\init_server.sqf";
```
Comment (Vdauphin) [inline =BTC=co@30_Hearts_and_Minds.Altis/init.sqf]: Just one indentation is necessary here

File: =BTC=co@30_Hearts_and_Minds.Altis/init.sqf [hunk 2]
```diff
@@ -2,19 +2,19 @@ enableSaving [false,false];
...
 };
 
-call compile preprocessFile "core\init_common.sqf";
+call compile preprocessFileLineNumbers "core\init_common.sqf";
 
 if (!isDedicated && hasInterface) then {
-	call compile preprocessFile "core\init_player.sqf";
+		call compile preprocessFileLineNumbers "core\init_player.sqf";
```
Comment (Vdauphin) [inline =BTC=co@30_Hearts_and_Minds.Altis/init.sqf]: Just one indentation is necessary here

File: =BTC=co@30_Hearts_and_Minds.Altis/init.sqf [hunk 3]
```diff
@@ -2,19 +2,19 @@ enableSaving [false,false];
...
 };
 
 if (!isDedicated && !hasInterface) then {
-	call compile preprocessFile "core\init_headless.sqf";
-};
\ No newline at end of file
+		call compile preprocessFileLineNumbers "core\init_headless.sqf";
```
Comment (Vdauphin) [inline =BTC=co@30_Hearts_and_Minds.Altis/init.sqf]: Just one indentation is necessary here

File: =BTC=co@30_Hearts_and_Minds.Altis/core/init_server.sqf [hunk 4]
```diff
@@ -1,23 +1,23 @@
...
 
 {[_x] spawn btc_fnc_task_create} foreach [0,1];
 
 if (btc_db_load && {profileNamespace getVariable [format ["btc_hm_%1_db",worldName],false]}) then {
-    if (btc_version isEqualTo (profileNamespace getVariable [format ["btc_hm_%1_version",worldName],1.13])) then    {
-        call compile preprocessFile "core\fnc\db\load.sqf";
-    } else {
-        call compile preprocessFile "core\fnc\db\load_old.sqf";
-    };
+	if (btc_version isEqualTo (profileNamespace getVariable [format ["btc_hm_%1_version",worldName],1.13])) then	{
```
Comment (Vdauphin) [inline =BTC=co@30_Hearts_and_Minds.Altis/core/init_server.sqf]: Change Tab for 4 spaces

File: =BTC=co@30_Hearts_and_Minds.Altis/core/init_server.sqf [hunk 5]
```diff
@@ -1,23 +1,23 @@
...
+		call compile preprocessFileLineNumbers "core\fnc\db\load.sqf";
```
Comment (Vdauphin) [inline =BTC=co@30_Hearts_and_Minds.Altis/core/init_server.sqf]: same

File: =BTC=co@30_Hearts_and_Minds.Altis/core/init_server.sqf [hunk 6]
```diff
@@ -1,23 +1,23 @@
...
+	} else {
```
Comment (Vdauphin) [inline =BTC=co@30_Hearts_and_Minds.Altis/core/init_server.sqf]: same

File: =BTC=co@30_Hearts_and_Minds.Altis/core/init_server.sqf [hunk 7]
```diff
@@ -1,23 +1,23 @@
...
+		call compile preprocessFileLineNumbers "core\fnc\db\load_old.sqf";
```
Comment (Vdauphin) [inline =BTC=co@30_Hearts_and_Minds.Altis/core/init_server.sqf]: same

File: =BTC=co@30_Hearts_and_Minds.Altis/core/init_server.sqf [hunk 8]
```diff
@@ -1,23 +1,23 @@
...
+	};
```
Comment (Vdauphin) [inline =BTC=co@30_Hearts_and_Minds.Altis/core/init_server.sqf]: same

File: =BTC=co@30_Hearts_and_Minds.Altis/core/init_server.sqf [hunk 9]
```diff
@@ -1,23 +1,23 @@
...
 } else {
     for "_i" from 1 to btc_hideout_n do {[] call btc_fnc_mil_create_hideout;};
 
     private _date = date;
     _date set [3, btc_p_time];
     setDate _date;
 
-    call compile preprocessFile "core\fnc\cache\init.sqf";
+	call compile preprocessFileLineNumbers "core\fnc\cache\init.sqf";
```
Comment (Vdauphin) [inline =BTC=co@30_Hearts_and_Minds.Altis/core/init_server.sqf]: same

File: =BTC=co@30_Hearts_and_Minds.Altis/core/init_server.sqf [hunk 10]
```diff
@@ -1,23 +1,23 @@
...
 
-    [] spawn {{waitUntil {!isNull _x}; _x call btc_fnc_db_add_veh;} foreach btc_vehicles;};
+	{[{!isNull _this},{_this call btc_fnc_db_add_veh;}, _x] call CBA_fnc_waitUntilAndExecute;} forEach btc_vehicles;
```
Comment (Vdauphin) [inline =BTC=co@30_Hearts_and_Minds.Altis/core/init_server.sqf]: same

File: =BTC=co@30_Hearts_and_Minds.Altis/init.sqf [hunk 11]
```diff
@@ -2,19 +2,19 @@ enableSaving [false,false];
...
 if (hasInterface) then {btc_intro_done = [] spawn btc_fnc_intro;};
 
-call compile preprocessFile "core\def\mission.sqf";
-call compile preprocessFile "define_mod.sqf";
+call compile preprocessFileLineNumbers "core\def\mission.sqf";
+call compile preprocessFileLineNumbers "define_mod.sqf";
 
 if (isServer) then {
-    call compile preprocessFile "core\init_server.sqf";
+	call compile preprocessFileLineNumbers "core\init_server.sqf";
```
Comment (Vdauphin) [inline =BTC=co@30_Hearts_and_Minds.Altis/init.sqf]: same

File: =BTC=co@30_Hearts_and_Minds.Altis/init.sqf [hunk 12]
```diff
@@ -2,19 +2,19 @@ enableSaving [false,false];
...
 };
 
-call compile preprocessFile "core\init_common.sqf";
+call compile preprocessFileLineNumbers "core\init_common.sqf";
 
 if (!isDedicated && hasInterface) then {
-    call compile preprocessFile "core\init_player.sqf";
+	call compile preprocessFileLineNumbers "core\init_player.sqf";
```
Comment (Vdauphin) [inline =BTC=co@30_Hearts_and_Minds.Altis/init.sqf]: same

File: =BTC=co@30_Hearts_and_Minds.Altis/init.sqf [hunk 13]
```diff
@@ -2,19 +2,19 @@ enableSaving [false,false];
...
 };
 
 if (!isDedicated && !hasInterface) then {
-    call compile preprocessFile "core\init_headless.sqf";
-};
\ No newline at end of file
+	call compile preprocessFileLineNumbers "core\init_headless.sqf";
```
Comment (Vdauphin) [inline =BTC=co@30_Hearts_and_Minds.Altis/init.sqf]: same

**Comment by 1kuemmel1** [comment]
> ...4 spaces instead of 1 tabulation...

Sorry, my mistake.
Corrected.

**Pull request merged**